server/schemas.js

```diff
@@ -35,12 +35,14 @@ const Authentication = mongoose.model('Authentication', authenticationSchema);
 
 const eventSchema = new mongoose.Schema({
     eventID: { type: String, required: true, unique: true},
-    eventName: { type: String, required: true},
-    location: { type: String },
-    collaborators: [{ type: String }],
-    viewers: [{ type: String }],
+    title: { type: String, required: true},
+    location: [ { address: String } ],
+    collaborators: [{ username: String, name: String }],
+    viewers: [{ username: String, name: String }],
```

Comment:
Can we also get the photoUrl, even if it's an empty string for viewers and collaborators?

Reply:
done!